extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use Email\EmailController;
+use Email\Fatal;
+
+abstract class FounderController extends EmailController {
```

Comment:
Blerg.  I kind of hate creating a class of constants that is just used to set other constants in different classes.  I can buy that ```WALL_NOTIFICATION``` is used more than once however so we would want to define it only once.  I'll move them over ...